components/suggestion/suggestion_box.jsx

```diff
@@ -690,24 +700,26 @@ export default class SuggestionBox extends React.Component {
                     onKeyDown={this.handleKeyDown}
                 />
                 {(this.props.openWhenEmpty || this.props.value.length >= this.props.requiredCharacters) && this.state.presentationType === 'text' &&
-                    <SuggestionListComponent
-                        ariaLiveRef={this.suggestionReadOut}
-                        open={this.state.focused}
-                        pretext={this.pretext}
-                        location={listStyle}
-                        renderDividers={renderDividers}
-                        renderNoResults={renderNoResults}
-                        onCompleteWord={this.handleCompleteWord}
-                        preventClose={this.preventSuggestionListClose}
-                        onItemHover={this.setSelection}
-                        cleared={this.state.cleared}
-                        matchedPretext={this.state.matchedPretext}
-                        items={this.state.items}
-                        terms={this.state.terms}
-                        selection={this.state.selection}
-                        components={this.state.components}
-                        wrapperHeight={this.props.wrapperHeight}
-                    />
+                    <div style={{position: 'fixed', zIndex: 101, width: this.state.width}}>
```

Comment:
This isn't going to work since it breaks the positioning of other autocompletes throughout the app (notably the ones in the post textbox and search box). To get around that, you could make another SuggestionList-like component for this case and use that instead.

If you do that, don't look at `SearchSuggestionList` when doing that though. It has `SearchSuggestionList extends` the base `SuggestionList` which causes some problems. You could instead make your new component just render a `SuggestionList` with a wrapper div around it instead.